lib/modules/customers/customers_page.dart

```diff
@@ -1,13 +1,97 @@
 import 'package:flutter/material.dart';
 
+import '../../../core/theme/app_theme.dart';
+
 class CustomersPage extends StatelessWidget {
   const CustomersPage({super.key});
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(title: const Text('Clientes')),
-      body: Container(),
+      body: Container(
+        padding: EdgeInsets.all(24),
+        child: Column(
+          spacing: 16,
+          mainAxisSize: MainAxisSize.max,
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: [
+            Align(
+              alignment: Alignment.centerRight,
+              child: OutlinedButton(
+                onPressed:
+                    (
+                      //context.goNamed('');
+                    ) {},
+                style: OutlinedButton.styleFrom(
+                  side: BorderSide(color: AppColors.darkPrimary300),
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(30),
+                  ),
+                  padding: const EdgeInsets.symmetric(
+                    horizontal: 20,
+                    vertical: 12,
+                  ),
+                ),
+                child: const Text(
+                  "Novo cliente",
+                  style: TextStyle(
+                    color: AppColors.darkPrimary300,
+                    fontSize: 16,
+                  ),
+                ),
+              ),
+            ),
+            CustomerInfo(name: "Vick", phone: "55 999271130"),
+            CustomerInfo(name: "Jorge", phone: "40028922"),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class CustomerInfo extends StatelessWidget {
+  const CustomerInfo({super.key, required this.name, required this.phone});
+
+  final String name;
+  final String phone;
+
+  String getInitials(String name) {
+    return name.isNotEmpty ? name[0].toUpperCase() : "?";
+  }
```

Comment:
Ajustando para um get que simplifica a visualização, quando for coisas pequenas e simples com a utilização de atributos já presentes da classe, pode fazer um get como este que fica show
```suggestion
  String get _initials => name.isNotEmpty ? name[0].toUpperCase() : '?';
```